library/test/Cargo.toml

```diff
@@ -9,4 +9,5 @@ std = { path = "../std" }
 core = { path = "../core" }
 
 [target.'cfg(not(all(windows, target_env = "msvc")))'.dependencies]
-libc = { version = "0.2.150", default-features = false }
+libc = { git = "https://github.com/thesummer/libc", rev = "2ada946801aec517051bb00cb83e890f9af5ad37", default-features = false, features = ['rustc-dep-of-std'], public = true  }
+# libc = { version = "0.2.150", default-features = false }
```

Comment:
Reminder that this needs to be put back before merge

Reply:
Thanks. Should be fixed now.